frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/pipeline-step-utils.ts

```diff
@@ -48,32 +48,32 @@ const getMatchingStepDuration = (matchingStep?: TaskStatusStep) => {
 export type StepStatus = {
   duration: string | null;
   name: string;
-  runStatus: runStatus;
+  ComputedStatus: ComputedStatus;
```

Comment:
This looks like a copy&paste and replaceAll mistake, the type is correct but we should keep a lowercase variable or attribute name here.

```suggestion
  status: ComputedStatus;
```

When changing this here, you need also update

1. `packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.tsx` and
2. `packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationStepList.tsx`